app/main.py

```diff
@@ -1,3 +1,63 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.trip_cost_calculation import trip_cost_calculation
+from app.customer import Customer
+from app.print_check import check_printer
+from app.shop import Shop
+from app.car import Car
+import json
+
+
+def shop_trip() -> None:
+    with open("app/config.json") as config:
+        config_dict = json.load(config)
+
+    fuel_price, customers_config, shops_config = config_dict.values()
+
+    customers_list = [
+        Customer(customer["name"],
+                 customer["product_cart"],
+                 customer["location"],
+                 customer["money"],
+                 Car(customer["car"]["brand"],
+                     customer["car"]["fuel_consumption"]))
+        for customer in customers_config
+    ]
+
+    shops_list = [
+        Shop(shop["name"], shop["location"], shop["products"])
+        for shop in shops_config
+    ]
```

Comment:
Use unpacking (using `**`)
Hint: You can place Car creation inside Customer init